core/services/gateway/connectionmanager.go

```diff
@@ -78,11 +79,12 @@ func NewConnectionManager(gwConfig *config.GatewayConfig, clock utils.Clock, lgg
 		}
 		nodes := make(map[string]*nodeState)
 		for _, nodeConfig := range donConfig.Members {
-			_, ok := nodes[nodeConfig.Address]
+			nodeAddress := strings.ToLower(nodeConfig.Address)
```

Comment:
I would prefer to deal with []byte, rather than converting lots of strings to lowercase. Not just here, but in the whole Gateway. Note that the Core uses `common.Address` (which is alias for [20]byte) for all comparisons. I believe all addresses (regardless of blockchain) should have representation as []byte.

Reply:
I agree that [20]byte might be better. I'll put it on a lower-pri backlog and we can address separately.
I hope that lowercasing needs to happen only when reading config and extracting signers from signatures.